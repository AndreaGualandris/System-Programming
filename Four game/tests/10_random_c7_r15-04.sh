#!/bin/bash
exec "$1" 7 15 < tests/10_random_c7_r15-04.txt
