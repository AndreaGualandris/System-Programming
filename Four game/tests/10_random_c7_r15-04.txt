0
6
1
2
3
0
5
0
6
0
3
3
3
4
3
3
5
4
2
3
1
0
0
3
1
2
5
3
5
4
4
2
4
2
3
1
3
1
2
4
0
3
4
4
3
0
0
4
6
2
6
0
6
2
5
5
0
6
0
5
6
4
1
1
3
3
0
4
3
3
6
0
6
2
3
0
6
0
0
0
5
4
4
5
2
6
4
2
4
0
2
6
6
4
2
3
5
3
2
2
2
6
4
3
4
