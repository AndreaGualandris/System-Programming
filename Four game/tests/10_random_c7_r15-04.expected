-------
       
       
       
       
       
       
       
       
       
       
       
       
       
       
       
-------
0123456
Player 'o' to move:
o
v
-------
       
       
       
       
       
       
       
       
       
       
       
       
       
       
o      
-------
0123456
Player 'x' to move:
      x
      v
-------
       
       
       
       
       
       
       
       
       
       
       
       
       
       
o     x
-------
0123456
Player 'o' to move:
 o
 v
-------
       
       
       
       
       
       
       
       
       
       
       
       
       
       
oo    x
-------
0123456
Player 'x' to move:
  x
  v
-------
       
       
       
       
       
       
       
       
       
       
       
       
       
       
oox   x
-------
0123456
Player 'o' to move:
   o
   v
-------
       
       
       
       
       
       
       
       
       
       
       
       
       
       
ooxo  x
-------
0123456
Player 'x' to move:
x
v
-------
       
       
       
       
       
       
       
       
       
       
       
       
       
x      
ooxo  x
-------
0123456
Player 'o' to move:
     o
     v
-------
       
       
       
       
       
       
       
       
       
       
       
       
       
x      
ooxo ox
-------
0123456
Player 'x' to move:
x
v
-------
       
       
       
       
       
       
       
       
       
       
       
       
x      
x      
ooxo ox
-------
0123456
Player 'o' to move:
      o
      v
-------
       
       
       
       
       
       
       
       
       
       
       
       
x      
x     o
ooxo ox
-------
0123456
Player 'x' to move:
x
v
-------
       
       
       
       
       
       
       
       
       
       
       
x      
x      
x     o
ooxo ox
-------
0123456
Player 'o' to move:
   o
   v
-------
       
       
       
       
       
       
       
       
       
       
       
x      
x      
x  o  o
ooxo ox
-------
0123456
Player 'x' to move:
   x
   v
-------
       
       
       
       
       
       
       
       
       
       
       
x      
x  x   
x  o  o
ooxo ox
-------
0123456
Player 'o' to move:
   o
   v
-------
       
       
       
       
       
       
       
       
       
       
       
x  o   
x  x   
x  o  o
ooxo ox
-------
0123456
Player 'x' to move:
    x
    v
-------
       
       
       
       
       
       
       
       
       
       
       
x  o   
x  x   
x  o  o
ooxoxox
-------
0123456
Player 'o' to move:
   o
   v
-------
       
       
       
       
       
       
       
       
       
       
   o   
x  o   
x  x   
x  o  o
ooxoxox
-------
0123456
Player 'x' to move:
   x
   v
-------
       
       
       
       
       
       
       
       
       
   x   
   o   
x  o   
x  x   
x  o  o
ooxoxox
-------
0123456
Player 'o' to move:
     o
     v
-------
       
       
       
       
       
       
       
       
       
   x   
   o   
x  o   
x  x   
x  o oo
ooxoxox
-------
0123456
Player 'x' to move:
    x
    v
-------
       
       
       
       
       
       
       
       
       
   x   
   o   
x  o   
x  x   
x  oxoo
ooxoxox
-------
0123456
Player 'o' to move:
  o
  v
-------
       
       
       
       
       
       
       
       
       
   x   
   o   
x  o   
x  x   
x ooxoo
ooxoxox
-------
0123456
Player 'x' to move:
   x
   v
-------
       
       
       
       
       
       
       
       
   x   
   x   
   o   
x  o   
x  x   
x ooxoo
ooxoxox
-------
0123456
Player 'o' to move:
 o
 v
-------
       
       
       
       
       
       
       
       
   x   
   x   
   o   
x  o   
x  x   
xoooxoo
ooxoxox
-------
0123456
Player 'x' to move:
x
v
-------
       
       
       
       
       
       
       
       
   x   
   x   
x  o   
x  o   
x  x   
xoooxoo
ooxoxox
-------
0123456
Player 'x' wins!
